domains/pics.json

```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "Vibred",
+           "email": "red@pissmail.com"
+        },
+    
+        "record": {
+            "CNAME": "telegraph-image2-c9p.pages.dev"
```

Comment:
You can't host an image cdn on is-a.dev, abuse risk.
is-a.dev is a service that allows developers to have free 'is-a.dev' subdomains for their personal websites.

Reply:
ok